testpilot/frontend/views.py

```diff
@@ -1,6 +1,12 @@
 from django.shortcuts import render
+from django.views.decorators.clickjacking import xframe_options_exempt
 
 
 def index(request, url=''):
     return render(request, 'testpilot/frontend/index.html',
                   {'user_id': request.user.email if not request.user.is_anonymous() else ''})
+
+
+@xframe_options_exempt
```

Comment:
Any chance we could use `parent-src` CSP around this to protect it from click-jacking? (IIRC `ALLOW-FROM` doesn't allow multiple origin values?)